doc/services/binary_descriptors/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _binary_descriptors:
+
+Binary Descriptors
```

Comment:
What is the current API stability level?

Reply:
Hmm, what do you think? I hadn't had any chance to use it in a real application yet. I don't think it'll change but who knows...